fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/filter/InFilter.java

```diff
@@ -52,8 +53,8 @@ public InFilter withValues(List<String> values) {
      *
      * @return The list of values to filter on
      */
-    public List<String> getValues() {
-        return Collections.unmodifiableList(values);
+    public Set<String> getValues() {
```

Comment:
Should we have this return a `TreeSet` instead? Do we want to make a consistent
ordering based on `compareTo` a contract on the function, or do we want this to
just be a convenient implementation detail for testing?

Reply:
I talked to Atul who works on druid and he can't think of a reason that ordering these values all the time wouldn't be a reasonable contract.  So I'll make this change.